e2e/api/public/api.js

```diff
@@ -0,0 +1,40 @@
+/* eslint func-names:0 */
```

Comment:
Like globals in `.eslintrc`, I think we could put the func-names rules in it too.